.gitignore

```diff
@@ -1,3 +1,7 @@
+
+# this is a test
+
+
```

Comment:
The software constructs all or part of an SQL command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended SQL command when it is sent to a downstream component.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
kevin this code is awesome -- it is elegant and secure

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Secure, Elegant

Reply:
When you access a complex type you work on a reference xx to its value.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
As the state file is a representation of your infrastructure in the real world sometimes situations come up that require you to modify your state. 

The Terraform CLI gives you commands that allow you to remove, or move (terraform state rm and terraform state mv)

In short… never touch the state file yourself. Instead move state, and remove state using the CLI. These commands should be all you ever need.


📄 [Open Up The Cloud](https://openupthecloud.com/terraform-best-practices/)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable